pyvrp/cpp/DurationSegment.h

```diff
@@ -32,24 +32,21 @@ namespace pyvrp
  *     Index of the last client in the route segment.
  * duration
  *     Total duration, including waiting time.
- * time_warp
- *     Total time warp on the route segment.
- * tw_early
+ * earliest_start
  *     Earliest visit moment of the first client.
- * tw_late
- *     Latest visit moment of the first client.
+ * latest_finish
+ *    Latest finish moment of the last client.
```

Comment:
Is this the time at which we depart from the last client in the segment, after having serviced them?

Reply:
Yes